### DISCUSSION TITLE
Feedback on the "Hello, world" tutorial

### DISCUSSION THREAD

**Discussion by konnov** [informalsystems/quint#509] [Polls]
We are experimenting with different forms of tutorials for Quint. If you have finished the tutorial, please let us know, which kind of the tutorial you like better: the CodeTour tutorial, the Markdown tutorial, or both.

If you have any further suggestions about improving the tutorial, please leave a comment below.

**Comment by otrack** [comment]
at step #9,  the UML diagram is not displayed
(vscode 1.77.3, ubuntu 20.04)

**Reply by NicolasRouquette** [reply]
Same here: vscode 1.88.1, ubuntu 22.04
